Modules/System Tests/Camera and Media Interaction/src/CameraTest.Codeunit.al

```diff
@@ -58,6 +58,32 @@ codeunit 135011 "Camera Test"
         Assert.AreEqual(CameraTestLibrary.GetSmallJpeg(), Base64Convert.ToBase64(PictureInStream), 'The mock picture was expected.');
     end;
 
+
+    [Test]
```

Comment:
Maybe a positive test that sets an acceptable quality value below 100 makes sense as well? As far as I can see, the Camera Page Impl. codeunit could benefit from some additional test coverage, and the only test in this codeunit that gets the picture, sets the default quality 100.

Reply:
I didn't know, how I could test this with the current test library..

I thought about adding another Base64 Image into the Test Library and returning this in the event `OnBeforeCameraInitialize` depending on the quality parameter.
Is this a good idea, or do have another idea how this could be achived?

Reply:
We'll be alright here...